admin/bulk-inactive-transfer.vbs

```diff
@@ -44,13 +44,18 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
-call changelog_update("02/14/2019", "Initial version.", "MiKayla Handley")
+CALL changelog_update("07/01/2022", "Update to ensure run is complete with error handling.", "MiKayla Handley, Hennepin County") '#868'
+CALL changelog_update("02/14/2019", "Initial version.", "MiKayla Handley, Hennepin County")
 
 'Actually displays the changelog. This function uses a text file located in the My Documents folder. It stores the name of the script file and a description of the most recent viewed change.
 changelog_display
 'END CHANGELOG BLOCK =======================================================================================================
-EMConnect ""
 '------------------------------------------------------------------------THE SCRIPT
+EMConnect ""
+'setting the footer month to make the updates in'
+back_to_self 'resetting MAXIS back to self before getting started
```

Comment:
Lines 55-57 - Put this after the dialog. If a user is passworded out, the script will not completed these actions.